notebooks/signac_201_Advanced_Indexing.ipynb

```diff
@@ -6,16 +6,34 @@
    "source": [
```

Comment:
<p>We should just delete this example IMHO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I recommend deferring all content updates for future PRs. Getting _something_ embedded is good for now, then we can iterate and rewrite.